VK_App/VK_App/ViewControllers/SecondBarVC/NewsTextCell.swift

```diff
@@ -7,11 +7,45 @@ import UIKit
 final class NewsTextCell: UITableViewCell, NewsConfigurable {
     // MARK: Private IBOutlets
 
-    @IBOutlet private var overviewPostLabels: UITextView!
+    @IBOutlet private var overviewPostLabel: UILabel!
+    @IBOutlet private var overviewButton: UIButton!
+
+    // MARK: Public properties
+
+    weak var delegate: NewsPostCellDelegate?
+    var isExpanded = false {
+        didSet {
+            updatePostLabel()
+            updateShowMoreOverview()
+        }
+    }
+
+    // MARK: Life cycle
+
+    override func awakeFromNib() {
+        super.awakeFromNib()
+        updatePostLabel()
+        updateShowMoreOverview()
+    }
 
     // MARK: Public Methods
 
     func configure(news: NewsFeed) {
-        overviewPostLabels.text = news.text
+        overviewPostLabel.text = news.text
+    }
+
+    // MARK: Private Methods
+
+    @IBAction private func buttonOverviewTapAction(_ sender: UIButton) {
+        delegate?.didTappedShowMore(self)
+    }
+
+    private func updatePostLabel() {
+        overviewPostLabel.numberOfLines = isExpanded ? 0 : 5
```

Comment:
```suggestion
        overviewPostLabel.numberOfLines = isExpanded ?  : 
```
Убираем магические числа